CDP4Dal/CDPMessageBus.cs

```diff
@@ -39,30 +39,17 @@ namespace CDP4Dal
     /// <summary>
     /// The CDP specific MessageBus implementation.
     /// </summary>
-    public class CDPMessageBus
+    public class CDPMessageBus : ICDPMessageBus, IDisposable

```

Comment:
moeten we deze niet decoraten met MEF attribute zodat het ook met MEF injected kan worden?

kijk ff naar de `DefaultPermissionProvider.cs` class voor inspiratie

Reply:
It could also be nice to specify the kind of registration to use for it ? Like "this class needs to be register as InstancePerLifetimeScope" ?